.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+jobs:
+  setup:
+    docker:
+      - image: node:10.13.0
```

Comment:
Any reason for this (old) version?

Reply:
Nothing specific, was just trying. Will change it to the one we have in package.json.

Reply:
changed to 10.15.3